config/plop/plopfile.mjs

```diff
@@ -14,9 +15,17 @@ export default function (plop) {
       .map((param) => param.slice(1, -1)) // remove brackets
       .filter((param) => param !== 'locale'); // remove locale param
   });
+  plop.setHelper('trailingParam', (value) =>
```

Comment:
I'm using this to find out if a developer creates a new file with a route that ends in a param, ie `[slug]`. Obviously that word might not be slug, but I left mapping that correctly as an exercise for the developer. I felt that asking how to handle a `trailingParam` that doesn't end in `[slug]` or remapping slug to the provided name would overcomplicate this functionality.